README.md

```diff
@@ -39,31 +96,42 @@ You can pass additional arguments to Theia after the image name, for example to
 
     # Linux, macOS, or PowerShell
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "$(pwd):/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
-    
+
     # Windows (cmd.exe)
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "%cd%:/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
 
-### Image Variants
+### Electron Apps
```

Comment:
Should we mention that "some assembly is required"? :) Something like:
"Build your own desktop Theia-based IDE using Electron" or such?